modules/nw-sriov-rdma-example-mellanox.adoc

```diff
@@ -108,7 +108,7 @@ spec:
     image: <RDMA_image> <2>
     securityContext:
```

Comment:
Please add "runAsUser: 0" under the securityContext

securityContext structure should look as follows: 
securityContext:
  runAsUser: 0
  capabilities:
    add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]